inc/httpRequest.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef HTTPREQUEST_HPP
+#define HTTPREQUEST_HPP
+
+#include <iostream>
+#include <map>
+#include <string.h>
+#include <string>
+#include <vector>
+
+class HttpRequest
+{
+  public:
+	HttpRequest();
+	virtual ~HttpRequest();
+
+	void parseBuffer(char *buff);
+	void printHttpReq();
+	bool methodIsAuthorized(std::string method) const;
+	// Getters
+	std::string getMethod() const;
+	std::string getPath() const;
+	std::string getProtocol() const;
+	std::string getHost() const;
+
+  private:
+	std::string						   trim(const std::string &s);
+	std::map<std::string, std::string> _request_map;
+	void							   parseFirstLine(std::string firstLine);
+	void							   parseOtherLines(std::vector<std::string> tmp_vector);
+};
+
+#endif
```

Comment:
comment the `endif` ?
```suggestion
#endif /* HTTPREQUEST_HPP */
```

Reply:
new line added at the end. 51d9b1bf71d1ae43deb91e00f0581784341ef02a
Can't .clang-format do that?

Reply:
Unfortunately, this feature is only available for the V16: [InsertNewlineAtEOF](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#insertnewlineateof).

My [comment](https://github.com/t-h2o/webserv/pull/21/files/0ed42f73c37332691fcb81f38876afe5deac4246#r1134470277) was talking about adding yes or no the comment `/* HTTPREQUEST_HPP */` on the `#endif`